app/jobs/email_suspended_members.rb

```diff
@@ -0,0 +1,19 @@
+class EmailSuspendedMembers
+  include Sidekiq::Worker
+  sidekiq_options queue: :members, retry: true, backtrace: true, failures: true 
+
+  def perform
+    suspended_members = Member.where(:status => "suspended")
```

Comment:
since we're using a newer version of ruby, hash keys can more readably be defined as `status: "suspended"`